src/languages/abnf.js

```diff
@@ -6,8 +6,8 @@ Category: syntax
 Audit: 2020
 */
 
-/** @type LanguageFn */
-export default function(hljs) {
+/** @type {LanguageFn} */
```

Comment:
I'll need to glance at the docs, but this looks much more acceptable.  But please remove all your other linting changes.